scripts/peek_dma.py

```diff
@@ -0,0 +1,50 @@
+def main():
+    DMA1_BASE = 0x40020000
+    DMA2_BASE = 0x40020400
+    base = DMA1_BASE
+
+    cl = cmdline.split()
```

Comment:
Which part, cmdline?
I pass that in with the environment when I call the script.  Look for the do_run function in the console debug utility and it will be clear.